concordium-consensus/src/Concordium/GlobalState/Basic/BlockState.hs

```diff
@@ -667,13 +667,25 @@ doGetUpdateKeysCollection :: (Monad m, HasBlockState s pv, IsProtocolVersion pv)
 doGetUpdateKeysCollection bs = return $! bs ^. blockUpdates . currentKeyCollection . unhashed
 
 doGetEnergyRate :: (Monad m, HasBlockState s pv) => s -> m EnergyRate
-doGetEnergyRate bs = return $! bs ^. blockUpdates . currentParameters . energyRate  
+doGetEnergyRate bs = return $! bs ^. blockUpdates . currentParameters . energyRate
+
+doGetEuroPerEnergy :: (Monad m, HasBlockState s pv) => s -> m ExchangeRate
+doGetEuroPerEnergy bs = return $! bs ^. blockUpdates . currentParameters . euroPerEnergy
+
+doGetAmountPerEuro :: (Monad m, HasBlockState s pv) => s -> m ExchangeRate
+doGetAmountPerEuro bs = return $! bs ^. blockUpdates . currentParameters . microGTUPerEuro
```

Comment:
There is a type called `ExchangeRates` which we should probably use instead of three separate functions.

It contains both the energy rate, euro per energy, and microGTUPerEuro.

Reply:
I have combined them into one now